src/collard-greens-with-smoked-duck-and-parnips.md

```diff
@@ -0,0 +1,30 @@
+# Collard Greens with Smoked Duck and Parsnips
+
+This dish was a pandemic left-over invention, loosely inspired by Ethiopian gomen. It can be served as a side dish, but we eat it an an entree. If you intend it as a side dish, then this serves 4. The smoked duck adds a bit of saltiness, so it's better to season to taste after cooking. Serve with good quality, crusty bread.
```

Comment:
```suggestion
This dish was a pandemic left-over invention, loosely inspired by Ethiopian gomen. It can be served as a side dish, but we eat it as an entree. If you intend it as a side dish, then this serves 4. The smoked duck adds a bit of saltiness, so it's better to season to taste after cooking. Serve with good quality, crusty bread.
```

Reply:
Thank you. I'm a noob, struggling with github a bit.